Docker-Swarm-deployment/single-node/monitoring-stack/mon-stack.yaml

```diff
@@ -104,7 +104,7 @@ services:
         tag: "{\"name\":\"{{.Name}}\",\"id\":\"{{.ID}}\"}"
 
   grafana:
-    image: ghcr.io/datakaveri/grafana:9.5.2
+    image: grafana/grafana:10.2.2
```

Comment:
We are building and maintaining a custom image for grafana in the datakaveri ghcr registry, as you can tell from previous image name. You can find the Dockerfile for it in the [grafana directory](https://github.com/datakaveri/iudx-deployment/tree/master/Docker-Swarm-deployment/single-node/monitoring-stack/grafana). Please update the grafana version there in the readme, build a new image using that base version and push to our ghcr registry which then can be used here.